README.md

```diff
@@ -1,43 +1,63 @@
 [![Build Status](https://github.com/espressif/esp-nuttx-bootloader/workflows/build/badge.svg)](https://github.com/espressif/esp-nuttx-bootloader/actions?query=branch%3Amain)
 
-# Bootloader and partition table for NuttX
+# Bootloader for NuttX
 
-This repository contains a minimal ESP-IDF project and build scripts used to produce 2nd stage bootloader and partition table binaries. Users of NuttX RTOS can download the binaries from release artifacts in this repository.
+This repository contains build scripts for producing the binaries for the 2nd stage bootloader. There are two bootloader options:
+- IDF bootloader (**default** option for NuttX images)
+- [MCUboot bootloader](https://github.com/mcu-tools/mcuboot/blob/main/docs/readme-espressif.md) (for loading NuttX images built with `CONFIG_ESP32_APP_FORMAT_MCUBOOT` option)
 
-# Obtaining the binaries
+Users of NuttX RTOS can download the binaries from release artifacts in this repository.
 
-## Downloading the latest version from Github
+# Downloading the latest version from Github
 
-Binaries built from the tip of the default branch or this repository can be obtained here:
+Binaries built from the tip of the default branch of this repository can be obtained here:
+
+## IDF bootloader
 
 Chip | Bootloader | Partition table
 -----|------------|-----------------
 ESP32 | [bootloader-esp32.bin](https://github.com/espressif/esp-nuttx-bootloader/releases/download/latest/bootloader-esp32.bin) | [partition-table-esp32.bin](https://github.com/espressif/esp-nuttx-bootloader/releases/download/latest/partition-table-esp32.bin)
 ESP32-S2 | [bootloader-esp32s2.bin](https://github.com/espressif/esp-nuttx-bootloader/releases/download/latest/bootloader-esp32s2.bin) | [partition-table-esp32s2.bin](https://github.com/espressif/esp-nuttx-bootloader/releases/download/latest/partition-table-esp32s2.bin)
 ESP32-C3 | [bootloader-esp32c3.bin](https://github.com/espressif/esp-nuttx-bootloader/releases/download/latest/bootloader-esp32c3.bin) | [partition-table-esp32c3.bin](https://github.com/espressif/esp-nuttx-bootloader/releases/download/latest/partition-table-esp32c3.bin)
+## MCUboot bootloader
+
+Chip | Bootloader
+-----|------------
+ESP32 | [mcuboot-esp32.bin](https://github.com/espressif/esp-nuttx-bootloader/releases/download/latest/mcuboot-esp32.bin)
+
+# Building locally
 
-## Building locally
+## IDF bootloader
 
-To build the binaries locally, it is recommended to run [build.sh](build.sh) script inside `espressif/idf` Docker image:
+It is recommended to build the binaries inside the `espressif/idf` Docker image.
 
 ```bash
-git clone https://github.com/espressif/esp-nuttx-bootloader.git
-cd esp-nuttx-bootloader
```

Comment:
I proposed to remove these instructions and just assume that the user had already done them.
The motivation for this is to avoid duplicating them on both `IDF bootloader` and `MCUboot bootloader` subsection.
An alternative would be to add a subsection for `Preparing the environment`, but I thought maybe it would be too obvious.

Reply:
I see.  I wanted to make sure that they weren't removed by mistake.